### ISSUE TITLE
[Feature Request] Add notification when app is trying to re-log in

### ISSUE THREAD

**Issue by migue802** [vrcx-team/VRCX#719]
**Explain in detail what your suggested feature would be used for.**
When the VRChat token expires for some reason, and the credentials are saved into the app, pop-up a notification on the system tray saying they need the 2FA code sent to the email.

**Describe how it would look if it requires a UI.**
It would not require an in-app UI, but rather a notification, maybe saying "Your account X session has expired, further action is required".

**Explain why people would want to use it.**
I kept receiving daily emails about someone trying to log in into my account on my email, and it scared the hell out of me because I thought someone had my password, but when I opened VRCX I noticed it was asking me to introduce the email 2FA code.
So basically: first, to not scare users who do not open the app daily but have it running on background; and second, to let know users that they will not receive notifications about this account because the session expired.

**Comment by SalbugVR** [comment]
This is a good idea. Honestly just having a notification informing you that your token expired would be great in general. I don't often notice VRCX needs to be relogged in for some time until I happen to recheck.

**Comment by Natsumi-sama** [comment]
I've gone with the simple solution of focusing the window whenever a email or 2FA code is required.

**Comment by migue802** [comment]
Haha nice, tysm! <3
